partner_industry_secondary/views/res_partner_view.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Antiun Ingenieria S.L. - Javier Iniesta
+     Copyright 2018 Eficent Business and IT Consulting Services, S.L. - Jordi Ballester
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+    <record model="ir.ui.view" id="view_partner_form_industry">
+        <field name="name">Partner form with industry</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="industry_id" position="after">
+                <field name="show_partner_industry_for_person" invisible="1" />
+                <field
+                    name="secondary_industry_ids"
+                    widget="many2many_tags"
+                    invisible="is_company == False and show_partner_industry_for_person == False"
```

Comment:
I imagine that the user should set this field after they set the field industry_id 1st. Because the this field got domain [('id', '!=', industry_id)]

So, we should hide this field also if industry_id = False

For example:
I set Partner A with industry_id then, display this field, so the user may think about their options on secondary_industry_ids

Do you agree with me?

Reply:
hello @DucTruongKomit 
Yes, your comment makes sense, but in this case the module already has a restriction in case the same industry is selected as main and secondary, only possible if a secondary industry is selected first.

![validationError](https://github.com/OCA/partner-contact/assets/89471929/1f7e808e-f270-4f7c-8b16-d7379ff94573)